udata_front/theme/gouvfr/assets/js/components/AdminTable/Table/AdminTable.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="fr-table">
```

Comment:
I think we can't scroll on the admin table horizontally in responsive mode.

Reply:
You're right, this is due to the `display: table` applied to have a full-width table on desktop. I'll have to change this based on the actual breakpoint

Reply:
The full-width table is now applied on LG breakpoint and wider